src/index.css

```
@@ -60,8 +60,7 @@ html {
 
 body {
 	/* color: var(--color-text); */
-	font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui,
-		helvetica neue, helvetica, Ubuntu, roboto, noto, arial, sans-serif;
+	font-family: 'Prompt', sans-serif;
```

```suggestion
	font-family: 'Pangolin', sans-serif;
```

You added a font named `Pangolin` and here you're referencing a different font. is this a mistake ?

---

No it is not a mistake. We are using different fonts for the logo and a different font for the body